addons/web/static/src/views/fields/iframe_wrapper/iframe_wrapper.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="web.IframeWrapperField" owl="1">
+        <div class="o_preview_iframe_wrapper">
```

Comment:
why the div? and the class `o_preview_iframe_wrapper`? and the class `o_preview_iframe`? it would be cool to onl have the iframe